### ISSUE TITLE
Webmin loses virtuall all "chrome"

### ISSUE THREAD

**Issue by FBachofner** [webmin/webmin#937]
Once in awhile Webmin will lose virtually all its "chrome" and I need to reinstall in order to get back full functionality.  This loss of "chrome" extends even to the login screen.

This has happened to me mostly on Ubuntu machines (typically 16.04.x) and much more rarely on LinuxMint (18.x).

This happened last night on an Ubuntu 16.04.4 server: I accidentally opened another website in the Firefox tab which had Webmin open in it.  I invoked the browsers "back" button and Webmin blew up.

Aspects of Webmin still "work" -- for example I can append "/session_login.cgi?logout=1" to the URL and logout occurs.  I can search for things using the search field and pressing enter.  Search results are displayed using the old theme, not "authentic" which is what I normally use for Webmin.

I initially feared Firefox might be at fault somehow -- perhaps not playing nicely with all the Webmin scripts, but I have tested this from other hosts and other browsers and all show the same corrupted Webmin pages.

Some questions:
1. Is there a way to avoid this problem?
2. Is there a way to fix this problem without reinstall?
3. Which logs might point to a source for the problem?

Thanks in advance!

I am attaching three screenshots to visually demonstrate the problem.
![webmin loses chrome login](https://user-images.githubusercontent.com/22858382/42650156-813f90e2-85c0-11e8-9a6b-076bed866a75.png)
![webmin loses chrome main page](https://user-images.githubusercontent.com/22858382/42650729-091ea754-85c2-11e8-98a1-328c5a5765f8.png)
![webmin loses chrome search results](https://user-images.githubusercontent.com/22858382/42650830-59f0fac4-85c2-11e8-87ff-60b081c8afed.png)

**Comment by rostovtsev** [comment]
Hi,

This happens due to the bug that is going to be fixed in upcoming Webmin release.

For now, all you can do, is to restore from backup `/etc/webmin/config` and restart Webmin. This is what actually happened here. Config file was destroyed.

**Comment by FBachofner** [comment]
Hi Ilia ( @iliaross ):

Thanks for the information.
Is there any bug report to which you can point me?

**Comment by rostovtsev** [comment]
There are few thread. Not all of them are related:

https://github.com/webmin/webmin/issues/755
https://www.virtualmin.com/node/54935
https://sourceforge.net/p/webadmin/bugs/5078/

**Comment by FBachofner** [comment]
Subsequent to this problem I purged and then reinstalled Webmin.

Now Webmin login form loads so slowly as to seem completely unresponsive.

Further, Webmin stops responding after navigating to a small handful of Webmin pages.

These problems manifest whether navigating to Webmin by the server's IP address or by (relevant) domain name.

This is on Ubuntu 16.04.  I updated (purge and reinstall) to Webmin 1.890 thinking that would improve the situation -- no benefit.  Argh!
